src/frontends/pytorch/src/utils.cpp

```diff
@@ -18,9 +18,9 @@ namespace frontend {
 namespace pytorch {
 
 void num_inputs_check(const NodeContext& context, size_t min_inputs, size_t max_inputs) {
-    auto inputs = context.inputs();
```

Comment:
If `context.inputs()` doesn't have this inlined_input now, it may invalidate the indices inside `context.inputs()` (e.g. operation that has first input inlined, but second input regular). I think better way would be instead of not adding it in `context.inputs()`, we could add empty `Output<Node>` there.

Reply:
Made the change as we discussed Maxim.